### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by kyam-mayden**
Styling added, some features removed to ensure completion.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
 	<title>Kyam Harris | Portfolio</title>
 	<link rel="stylesheet" type="text/css" href="normalize.css">
 	<link rel="stylesheet" type="text/css" href="styles.css">
+	<link href="https://fonts.googleapis.com/css?family=Roboto:200,400,500,700" rel="stylesheet">
+
```
Comment (karimkefi) [inline index.html]: remove space

File: index.html [hunk 2]
```diff
@@ -87,46 +89,21 @@ <h3><a href="#">Academy logo</a></h3>
...
-						purus condimentum quis. Suspendisse lectus tortor, volutpat sit
-						amet pretium vel, tincidunt at est. Pellentesque tincidunt orci
-						at porta malesuada. Morbi turpis elit, viverra vel tempus nec</p>
-				</section>
-			</article>
 		</div>
 	</main>
 	<footer class="container" id="foot">
-		CONTACT DETAILS
+		<h4> id="email">email: kyam.lh@googlemail.com</h4>
```
Comment (karimkefi) [inline index.html]: looks like there are 2 closing tags h4> ... email">

File: index.html (hunk 1 above)
Comment (karimkefi) [inline index.html]: remove line

File: styles.css [hunk 3]
```diff
@@ -1,25 +1,26 @@
 *{
 	box-sizing:border-box;
+	font-family:Roboto;
```
Comment (karimkefi) [inline styles.css]: you might want to add a fallback font

File: styles.css [hunk 4]
```diff
@@ -74,13 +81,12 @@ a{
 }
 
 .navlinks,
-.headImage{
+.headImage {
```
Comment (karimkefi) [inline styles.css]: remove space before {

File: styles.css [hunk 5]
```diff
@@ -92,31 +98,72 @@ a{
...
 .buttonHolder div h2{
 	text-align:center;
 	height:100%;
 	position:relative;
->>>>>>> responsive
+}
+
+.buttonHolder div:hover{
+	background-color:#999999;
+
```
Comment (karimkefi) [inline styles.css]: remove empty line

File: styles.css [hunk 6]
```diff
@@ -167,17 +215,19 @@ div[class^="logo-"] img{
 
 /*content*/
 
-main.container{
-	min-height:1200px;
-	margin-top:20px;
+main.container {
```
Comment (karimkefi) [inline styles.css]: remove space before {

File: styles.css [hunk 7]
```diff
@@ -230,31 +291,50 @@ article{
 	bottom:0;
 }
 
+.item3 .itemText{
```
Comment (karimkefi) [inline styles.css]: "item3" not very semantic :)

File: styles.css [hunk 8]
```diff
@@ -230,31 +291,50 @@ article{
...
-@media screen and (max-width:1250px){
-	.itemPic{
-		width:85%;
-	}
+#phone{
+	float:right;
+	margin:0;
 }
 
+/*Media Queries*/
```
Comment (karimkefi) [inline styles.css:315]: remove this section if not doing anything

File: styles.css [hunk 9]
```diff
@@ -230,31 +291,50 @@ article{
...
 #foot{
-	height:100px;
+	height:50px;
 	border:3px solid #000000;
 	padding:5px;
 	margin-top:20px;
 }
 
-/*Media Queries*/
+#email{
```
Comment (karimkefi) [inline styles.css:305]: appears to be in the media query section from comment, but not under the @media screen section

**Review by karimkefi — CHANGES_REQUESTED** [review]
few formatting items and comments on media queries

File: index.html (hunk 1 above)
Comment (kyam-mayden) [reply index.html]: Resolved

File: styles.css (hunk 3 above)
Comment (kyam-mayden) [reply styles.css]: resolved

File: styles.css (hunk 4 above)
Comment (kyam-mayden) [reply styles.css]: Resolved

File: styles.css (hunk 5 above)
Comment (kyam-mayden) [reply styles.css]: Resolved

File: styles.css (hunk 6 above)
Comment (kyam-mayden) [reply styles.css]: Resolved

File: styles.css (hunk 8 above)
Comment (kyam-mayden) [reply styles.css:315]: Resolved

File: styles.css (hunk 9 above)
Comment (kyam-mayden) [reply styles.css:305]: Resolved

File: styles.css (hunk 7 above)
Comment (kyam-mayden) [reply styles.css]: Resolved - item#'s removed

File: index.html [hunk 10]
```diff
@@ -5,7 +5,6 @@
 	<link rel="stylesheet" type="text/css" href="normalize.css">
 	<link rel="stylesheet" type="text/css" href="styles.css">
 	<link href="https://fonts.googleapis.com/css?family=Roboto:200,400,500,700" rel="stylesheet">
-
 </head>
```
Comment (karimkefi) [inline index.html]: just to warn you. Mike asked me to remove all empty lines throughout the html.

**Review by karimkefi — APPROVED** [review]

File: styles.css [hunk 11]
```diff
@@ -345,7 +336,7 @@ article{
 		display:none;
 	}
 
-	.item1{
+	.PrimaryPFItem{
 		width:100%;
```
Comment (karimkefi) [inline styles.css:338]: you could align the indentations throughout to make it consistent

**Review by karimkefi — APPROVED** [review]

File: index.html [hunk 12]
```diff
@@ -28,16 +28,16 @@ <h4>Somerset based Web Developer</h4>
 			<div class="headImage">
 				<div class="buttonHolder">
 					<div>
-						<a href="#bigBox"><h2>View Portfolio</h2></a>
-
+						<a href="#bigBox"><h2>View Portfolio</h2>
```
Comment (karimkefi) [inline index.html]: < h2 > should be indented on a new line

Comment (kyam-mayden) [reply index.html]: Resolved

**Review by karimkefi — APPROVED** [review]

**Pull request merged**